packages/loader/container-loader/src/container.ts

```diff
@@ -393,8 +397,8 @@ export class Container extends EventEmitterWithErrorHandling<IContainerEvents> i
     private _attachState = AttachState.Detached;
 
     // Active chaincode and associated runtime
-    private _storageService: IDocumentStorageService | undefined;
-    private get storageService() {
+    private _storageService: IDocumentStorageService & IDisposable | undefined;
```

Comment:
Github ate my comment :-/

Do you think `IDocumentStorageService` should extend `IDisposable`?  Or alternatively, is there benefit to this typing as opposed to just typing it as `RetriableDocumentStorageService` like `DeltaManager` does today?

Reply:
There is already a bug for it, and yes i do: #5980 . My change is a bit pre-emptive, in #5902 i'll wrap `RetriableDocumentStorageService`  with the protocol tree handler

Reply:
Started the work for this change in #8083